packages/sdk/src/restclient.spec.ts

```diff
@@ -223,6 +231,7 @@ describe("RestClient", () => {
       });
     });
 
+    // this is failing for me on first run (faucet has not signed anything)
```

Comment:
True. I considered this unproblematic as deployment of contracts should be part of the default chain start flow soon.